values.md

```diff
@@ -32,7 +32,7 @@ While we believe that we are building something exceptional, that knowledge does
 
 ### We optimize for the long-term.
 
-While we pursue the mission with intense focus, we recognize that creating the change in the world that we wish to see will not happen overnight. We think, plan, and act for the long-term.
+We recognize that creating the change in the world that we wish to see will not happen overnight. Earning the right to pursue this work over decades requires both long-term vision and near-term execution, both user love and commercial success.
```

Comment:
> commercial success.

This phrase has connotations associated with it that may be helpful to clarify.

1. Success to what degree? How much success is "good enough" to fuel our mission? Perhaps this success can be tied back to the section currently being updated on assigning value to each product we create.

2. It may be helpful to spell out the "whys" embedded in these statements: Why is it important to have user love if you already have commercial success? What happens if you only have one or the other?

Reply:
Having the value reading "We optimize for the long-term" but the description balancing long/short in equal parts may cause a little dissonance. 

Perhaps "Earning the right to pursue a long-term vision over decades requires near-term execution on both user love and commercial success"?